commcare_connect/reports/tables.py

```diff
@@ -1,38 +1,42 @@
+import calendar
+
 from django.urls import reverse
 from django.utils.html import format_html
 from django_tables2 import columns, tables
 
+from commcare_connect.opportunity.tables import SumColumn
+
 
 class AdminReportTable(tables.Table):
-    quarter = columns.Column(verbose_name="Quarter")
+    month = columns.Column(verbose_name="Month", footer="Total")
     delivery_type = columns.Column(verbose_name="Delivery Type")
-    users = columns.Column(verbose_name="Active Users")
-    services = columns.Column(verbose_name="Verified Services")
-    approved_payments = columns.Column(verbose_name="Acknowledged Payments")
-    total_payments = columns.Column(verbose_name="Total Payments")
-    beneficiaries = columns.Column(verbose_name="Beneficiaries Served")
+    users = SumColumn(verbose_name="Active Users")
```

Comment:
Jon wants us to change this name. I propose "Eligible Users" for now. We can always change it

Reply:
Done. [ce5f297](https://github.com/dimagi/commcare-connect/pull/485/commits/ce5f2972b3a3da59e2d769f4036a0e6c04a43531)